tests/foreman/api/test_setting.py

```diff
@@ -17,13 +17,18 @@
 """
 from nailgun import entities
 from robottelo.datafactory import generate_strings_list, valid_data_list
-from robottelo.decorators import run_in_one_thread, tier1, upgrade
+from robottelo.decorators import run_in_one_thread, stubbed, tier1, upgrade
 from robottelo.test import APITestCase
 
 
 class SettingTestCase(APITestCase):
     """Implements tests for Settings for API"""
 
+    def setUp(self):
+        """Initialize test variables"""
+        super(SettingTestCase, self).setUp()
+        self.original_value = None
```

Comment:
Why do you need create a None var on setup?

Reply:
@renzon original_value is used for reverting the value of parameter so that it should not affect other tests. It will be none in setup and before test starts, original value will be set.